dashboard/app/dsl/script_dsl.rb

```diff
@@ -330,6 +333,7 @@ def self.serialize_to_string(script)
     s << "id '#{legacy_script_ids[script.name]}'" if legacy_script_ids[script.name]
 
     s << "professional_learning_course '#{script.professional_learning_course}'" if script.professional_learning_course
+    s << "only_instructor_review_required #{script.only_instructor_review_required}" if script.only_instructor_review_required
```

Comment:
For my own learning, what is this dsl class for?

Reply:
I don't 100% understand the details here, but generally this is how we keep track of script definitions and sync script/level updates from levelbuilder. The script I was playing with while working on this produced changes to these files (first one uses the DSL format, second one has more detail and is JSON?):

https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/config/scripts/csp4-dlp-18.script
https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/config/scripts_json/csp4-dlp-18.script_json

Reply:
Script DSL is the way we serialize all the information about older scripts.

Reply:
Good to know, thanks Dani! Is it being deprecated in favor of the JSON format?

Reply:
Yup! We are trying to move everything over to the JSON format.  We will probably accomplish this for script soonish. There are still levels that use DSL which will be around longer